CHANGELOG.md

```diff
@@ -6,6 +6,17 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [16.0.0]
+### Changed
+- **BREAKING**: Remove `syncTokens` method from the TokenListController ([#590](https://github.com/MetaMask/controllers/pull/590))
+  - This is breaking for any consumers of this method.
+- Enable default caching for token-service ([#594](https://github.com/MetaMask/controllers/pull/594))
```

Comment:
Updated to use a more easily understood term for the style of caching we're now using, and to reference the name of the thing being cached rather than the internal API.
```suggestion
- Enable HTTP caching for the dynamic token list managed by the TokenListController ([#594](https://github.com/MetaMask/controllers/pull/594))
```